upstairs/src/lib.rs

```diff
@@ -5028,7 +5762,7 @@ impl BlockIO for Guest {
          * XXX Figure out how long to wait for this.  The time to go active
          * will include the time to reconcile all three downstairs.
          */
-        for _ in 0..10 {
+        for _ in 0..1000 {
```

Comment:
This is the hackery that I need to fix.
Sometimes an activation can take several minutes (because I'm restarting downstairs every 3 seconds).
I'll probably put this back how it was and have a follow on PR to redo this and take the whole guest local active idea out of here.

Reply:
removed.  Issue filed